index.html

```diff
@@ -942,8 +941,10 @@ <h1><dfn>Coalesced and predicted events</dfn></h1>
             <h2><dfn>Coalesced events</dfn></h2>
 
             <p>For performance reasons, user agents may choose not to send a <code>pointermove</code>
-            event every time the position of a pointer is updated. Instead, they will coalesce
-            (combine/merge) multiple position changes into a single move event. While
+            event every time a <a data-lt="measurable properties">measurable property</a>
+            (such as coordinates, pressure, tangential pressure, tilt, twist, or contact geometry)
+            of a pointer is updated. Instead, they will coalesce (combine/merge) multiple changes into
```

Comment:
I think this should say "Instead, they may coalesce".

Reply:
good point yes, that would gel with the "may" in the previous sentence. will make that change in a sec.